docs/getting_started/aws.rst

```diff
@@ -27,14 +27,27 @@ the template located at ``terraform/aws.sample.tf``. The beginning of it looks l
 
   provider "aws" {
     access_key = ""
-    secrect_key = ""
+    secret_key = ""
     region = "${var.region}"
   }
 
-Copy that file in it's entirety to the root of the project as ``aws.tf`` to start
+Copy that *file* in it's entirety to the root of the project as ``aws.tf`` to start
 customization. In the next sections, we'll describe the settings that you need
 to configure.
 
+Do not copy the text contents above into a file, if you do not have the terraform/aws.sample.tf file, you need to clone the mantl repository.
+Please note, newer versions of this file do not have "access_key" or "secret_key" lines, we automatically find your AWS credentials from amazon's new "AWS Credentials file" standard.
```

Comment:
Can you explain this in more detail? We should also probably just do a RST include of the aws.tf file, instead of manually copying its contents.